core/modules/layout/layout.admin.inc

```diff
@@ -885,6 +899,78 @@ function layout_settings_form_validate(&$form, &$form_state) {
       form_error($form['conditions']['active'][$key], t('The condition \'!condition\' does not have the required contexts at this path. Remove the condition to save the layout.', array('!condition' => $condition->summary())));
     }
   }
+
+  global $_layout_additional_paths;
+  $_layout_additional_paths = array();
+  if (isset($form_state['values']['additional_paths'])) {
+    $primary_path = $form_state['values']['path'];
+    $num_placeholders = count(_layout_placeholder_positions($primary_path));
+    $paths = explode("\n", $form_state['values']['additional_paths']);
+    foreach ($paths as $path){
+      $path = trim($path);
+      if (!empty($path)) {
+        $layout_provides_path = layout_provides_path($path);
+        $router_item = menu_get_item($path);
+        $router_item_path = $router_item ? (string) $router_item['path'] : '';
+        $num_additional_placeholders = count(_layout_placeholder_positions($path));
+        $alias = NULL;
+        if (strpos($path, '%') === FALSE) {
+          $alias = db_query('SELECT * FROM {url_alias} WHERE alias = :path', array(':path' => $path))->fetchObject();
+        }
+        switch (TRUE) {
+          case ($num_placeholders != $num_additional_placeholders):
+            // E.g. Cannot have "node/%" and "node".
```

Comment:
Use of `e.g.` and `i.e.` have a complicated history in Drupal. Can we not use an abbreviation here and just use `Example:`

```suggestion
            // Example: Cannot have "node/%" and "node".
```

Same for the several code comments below.

Reply:
I would use *For example, it cannot have "node/%" nor "node".*